kafka/kafka-2.8.0/image.yaml

```diff
@@ -51,7 +55,6 @@ packages:
     - bind-utils
     - tini
     - lsof
-    - kafka_exporter
```

Comment:
Since the kafka_exporter is still installed via the module.yaml, we can remove it from here